fairscale/experimental/wgit/compression.py

```diff
@@ -0,0 +1,150 @@
+from collections import OrderedDict
```

Comment:
don't forget the copyright header too. I will take a look when this is ready. Add simple tests with fixed tensors and expected golden values to ensure the coverage for the new code here. Thanks!